dart/test/sentry_client_test.dart

```diff
@@ -396,6 +396,71 @@ void main() {
     });
   });
 
+  group('SentryClient : apply default pii', () {
+    test('sendDefaultPii is disabled', () async {
+      var options = SentryOptions(dsn: fakeDsn);
+      options.sendDefaultPii = false;
+      final transport = MockTransport();
+      options.transport = transport;
+      final client = SentryClient(options);
```

Comment:
I'd prefer to use a Fixture pattern, so we can avoid all this code duplication within the tests, which might change only the value of `sendDefaultPii`, an example would be: `file_system_transport_test.dart` or the Java source code eg https://github.com/getsentry/sentry-java/blob/main/sentry/src/test/java/io/sentry/DiagnosticLoggerTest.kt

Reply:
Makes sense 👍